runbotics/runbotics-orchestrator-ui/src/src-app/components/guards/GuestGuard.tsx

```diff
@@ -18,7 +19,11 @@ export const withGuestGuard = (Component: FC | VFC) => (props: any) => {
         switchLanguage(router.locale as Language);
     }, [router.locale] );
 
-    if (isBrowser && isInitialised && isAuthenticated) router.replace('/app/processes');
+    if (isBrowser && isInitialised && isAuthenticated) {
+        const destinationPage = StorageManagerService.readDestinationPage();
+        if (destinationPage) router.replace(destinationPage);
+        StorageManagerService.removeDestinationPage();
```

Comment:
when destinationPage is empty .removeDestinationPage(); does not cause an error?